internal/pkg/constants/constants.go

```diff
@@ -68,6 +68,24 @@ const (
 	// KubeadmCAKey is the path to the root CA private key.
 	KubeadmCAKey = "/etc/kubernetes/pki/ca.key"
 
+	// KubeadmSACert is the path to the SA certificate.
+	KubeadmSACert = "/etc/kubernetes/pki/sa.pub"
```

Comment:
I know the expectations were already set because we currently fine `KubeadmCAKey` here and you were following suit, but I'm wondering if we should use the constants from `kubeadm`: https://github.com/kubernetes/kubernetes/blob/a3fd657d90a6bc091446f5e1744e34c1ab152489/cmd/kubeadm/app/constants/constants.go?